### ISSUE TITLE
Investigate using ProseMirror for markdown

### ISSUE THREAD

**Issue by jasongrout** [jupyterlab/jupyterlab#5908]
It would be interesting to use ProseMirror for Markdown editing: http://prosemirror.net/examples/markdown/

**Comment by ian-r-rose** [comment]
I agree. @bollwyvl made a proof-of-concept for this a few months ago.

**Comment by saulshanabrook** [comment]
https://github.com/jupytercalpoly/jupyterlab-richtext-mode

**Comment by gutow** [comment]
Also consider this project https://github.com/g2nb/jupyter-wysiwyg, which is partway there. It does not deal well with embeded latex expressions and still needs interface work, but basically does what is needed.
